tests/test_pois.py

```diff
@@ -0,0 +1,274 @@
+import os
+import unittest
+from flask import jsonify
+from api import app, db
+from flask import Flask, request
+from api.models import POI, Media, Link, Story, StoryPOI
+import requests
+from flask import jsonify
+import json
+from datetime import date
+
+#Complete params
+
+poi_create_complete = {
+    'name': 'Morrow Plots',
+    'date': '2018-03-05',
+    'description': 'Just corn here',
+    'map_year': 1856,
+    'x_coord': 76,
+    'y_coord': 42,
+    'links': [
+        {
+            'link_url': 'http://fbnew.com',
+            'display_name': 'FacebookNew'
+        }
+    ],
+    'media': [
+        {
+            'content_url': 'http://images.com/alpaca.jpg',
+            'caption': 'The new alpaca in its natural habitat'
+        }
+    ],
+    'story_ids': [1, 2]
+}
+
+poi_update_complete = {
+    'name': 'Morrow Plots',
+    'date': '2018-03-05',
+    'description': 'Just corn here',
+    'links': [
+        {
+            'link_url': 'http://fbnew.com',
+            'display_name': 'FacebookNew'
+        }
+    ],
+    'media': [
+        {
+            'content_url': 'http://images.com/alpaca.jpg',
+            'caption': 'The new alpaca in its natural habitat'
+        }
+    ],
+    'story_ids': [1, 2]
+}
+
+#Permissible (only required attributes, optional ones not included)
+
+poi_create_permissible = {
+    #This one needs EITHER map_year OR story_ids
+    'name': 'Morrow Plots',
+    'date': '1856-01-01',
+    'description': 'Just corn here',
+    'map_year': 1856,
+    'x_coord': 76,
+    'y_coord': 42,
+    'links': [],
+    'media': [],
+    'story_ids': []
+}
+
+poi_edit_permissible = {
+    'name': 'Morrow Plots',
+    #Technically date is not required, but frontend will automatically send in date as 1/1/mapyear
+    #Otherwise there would be an error when running this test
+    'date': '1856-01-01',
+    'description': 'Just corn here',
+    'links': [],
+    'media': [],
+    'story_ids': []
+}
```

Comment:
technically all the poi fields are optional for PUT requests: [docs](https://github.com/hack4impact-uiuc/nnb-backend-v2/blob/master/backend-v2.md#endpoint-3). 

you probably dont even need this since the `poi_update_complete` test will cover everything this would cover